wgengine/tstun/tun.go

```diff
@@ -136,8 +137,9 @@ func WrapTUN(logf logger.Logf, tdev tun.Device) *TUN {
 // destination (the map keys).
 //
 // The map ownership passes to the TUN. It must be non-nil.
-func (t *TUN) SetDestIPActivityFuncs(m map[packet.IP4]func()) {
-	t.destIPActivity.Store(m)
+func (t *TUN) SetDestIPActivityFuncs(m4 map[packet.IP4]func(), m6 map[packet.IP6]func()) {
```

Comment:
If we did https://github.com/inetaf/netaddr/issues/53 first, could this simplify down to just one map again?